### PR TITLE
Increase seed size to int64

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

File: dali/pipeline/pipeline.h [hunk 1]
```diff
@@ -300,7 +300,7 @@ class DLL_PUBLIC Pipeline {
     DALI_ENFORCE(batch_size_ > 0, "Batch size must be greater than 0");
     seed_.resize(MAX_SEEDS);
     current_seed_ = 0;
-    if (seed != -1) {
+    if (seed >= -1) {
```
Comment (ptrendx) [inline dali/pipeline/pipeline.h]: > not >=

Comment (JanuszL) [reply dali/pipeline/pipeline.h]: My bad

**Comment by JanuszL** [comment]
The question is if anyone saves DALI serialized pipeline. But we will add this to our docs.

File: dali/pipeline/pipeline.h (hunk 1 above)
Comment (JanuszL) [reply dali/pipeline/pipeline.h]: Done

**Comment by JanuszL** [comment]
Rebuilds 38233.

**Pull request merged**